root/layout.tt

```diff
@@ -103,6 +103,7 @@
                 <a href="https://metabrainz.org/donate" class="internal">[% l('Donate') %]</a> |
                 <a href="//wiki.musicbrainz.org/" class="internal">[% l('Wiki') %]</a> |
                 <a href="https://community.metabrainz.org/" class="internal">[% l('Forums') %]</a> |
+                <a href="/doc/Communication/IRC" class="internal">[% l('IRC') %]</a> |
```

Comment:
I think I’d prefer "Chat" over "IRC", FWIW.

Reply:
There is only an “IRC” doc page for now. When there will be another doc page about another tool, e.g. Telegram , or a “Chat” doc page to carry both, then this link can be changed.

Reply:
We don’t have a doc page for "Forums" either, but we still don’t call the link "community" (the domain) nor do we call it "Discourse" (the technology/platform). The "bug tracker" link is also is also not called "tickets" or "JIRA".

IRC is the technology/platform, but what we do there is chat.

Also see https://community.metabrainz.org/t/idea-rename-irc-to-chat/450955?u=freso

Reply:
Thanks for having opened ticket MBS-10470 that can be used for follow-up.
I’m closing this conversation as this PR is already merged and forums/tickets are preferred.